src/screens/EditPlaylistScreen.tsx

```diff
@@ -35,7 +37,27 @@ export class EditPlaylistScreen extends React.Component<Props, State> {
   static navigationOptions = ({ navigation }) => {
     return {
       title: 'Edit Playlist',
-      headerRight: <NavHeaderButtonText handlePress={navigation.getParam('updatePlaylist')} text='Save' />
+      headerRight: (
```

Comment:
I think here you should extract everything you need from navigation.params and only render the RNView once and pass it the props it should use. Instead of conditionally rendering the rnviews